### ISSUE TITLE
Not able to read Commodore 8032 tape

### ISSUE THREAD

**Issue by ThomasHeinrichSchmidt** [eightbitjim/cassette-nibbler#44]
I was not able to read a Commodore 8032 tape cassette.
I tried 
```
java -jar cassette-nibbler-0.2.jar -platform=commodore -logging=verbose "SIDE 2.wav" 
```
and also using other platforms like `commodore16+4` or `c64-turbotape` but with no luck.
I only got error messages like these:
```
(1927029442) FRAME ERROR
(1927029442) OPTIONS SPECIFY TO ALLOW CHECKBIT ERRORS, SO WILL TREAT FRAME AS VALID
(1927029442) INVALID SYNC BYTE
(1927029442) OPTIONS SPECIFIED TO ATTEMPT TO RECOVER FILES. ATTEMPTING TO RECOVER THIS BLOCK. 
...
```
or
```
(0) INVALID FRAME START. PULSE LENGTHS 0, 113378
(0) UNEXPECTED PULSE IN FRAME
(0) INVALID FRAME START. PULSE LENGTHS 204081, 997733
(0) UNEXPECTED PULSE IN FRAME
(0) INVALID FRAME START. PULSE LENGTHS 90703, 181406
(0) UNEXPECTED PULSE IN FRAME
(0) CHECKBIT ERROR IN FRAME 
...
```

and nothing was saved.
I enclose the WAV file in question (had to split into 10MB chunks "SIDE 2.z0#" , also enclosed the batch to  rename them to "SIDE 2-0#.zip" to make GitHub happy. You have to re-rename them, I guess, before beeing able to acces the "SIDE 2.zip").
[SIDE 2.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612327/SIDE.2.zip)
[SIDE 2-01.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612330/SIDE.2-01.zip)
[SIDE 2-02.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612331/SIDE.2-02.zip)
[SIDE 2-03.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612332/SIDE.2-03.zip)
[SIDE 2-04.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612333/SIDE.2-04.zip)
[SIDE 2-05.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612334/SIDE.2-05.zip)
[SIDE 2-06.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612335/SIDE.2-06.zip)
[SIDE 2-07.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612336/SIDE.2-07.zip)
[SIDE 2-08.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612337/SIDE.2-08.zip)
[SIDE 2-09.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612338/SIDE.2-09.zip)

[tozip.zip](https://github.com/eightbitjim/cassette-nibbler/files/2612339/tozip.zip)

**Comment by eightbitjim** [comment]
Try using the -invert option. I just tried it and apparently got 3 files out.

**Comment by ThomasHeinrichSchmidt** [comment]
Wow! That's way better. Yes, I do also expect three files (from the toc I noted on the tape cassette label). And cassette-nibbler also tells me that it is extracting three files:
```
Progress-----------------------------------------------------------------------
100%: SIDE 2.wav (0 files extracted): SIDE 2.wav
100%: SIDE 2.wav (1 file extracted): SIDE 2.wav
100%: SIDE 2.wav (2 files extracted): SIDE 2.wav
100%: SIDE 2.wav (3 files extracted): SIDE 2.wav
100%: Processed 21 minutes 26 seconds (87.0x realtime) (3 files extracted): Pro
```
But I do actually receive only two files `3dmess.prg.ROMloader.e1783388.commodore.PRG` and `"3dmessv 2.0.prg.ROMloader.98108ece.commodore.PRG"` . From my notes I expect another file `"3dmessv2.0.prg.ROMloader.98108ece.commodore.PRG"` (note the missing space before the 2.0). Maybe cassette-nibbler could display the names of the files it's extracting and why some file does not get extracted in the end ...
But otherwise great work! I had never expected to get those files back ...

**Comment by eightbitjim** [comment]
Great, glad that worked. Thank you for the comments-- I'll see what extra information I can get it to display.

I got 3 files out successfully, but one had a colon in the filename. I wonder if this is another difference between Linux and Windows... perhaps the file with the colon in didn't work on your machine. I've uploaded a new version that strips the colons from filenames if they occur. This might be more successful, hopefully.
